sysdump/constants.go

```diff
@@ -10,19 +10,20 @@ import (
 )
 
 const (
-	awsNodeDaemonSetName         = "aws-node"
-	awsNodeDaemonSetNamespace    = "kube-system"
-	ciliumAgentContainerName     = "cilium-agent"
-	ciliumConfigConfigMapName    = "cilium-config"
-	ciliumDaemonSetName          = "cilium"
-	ciliumEtcdSecretsSecretName  = "cilium-etcd-secrets"
-	ciliumOperatorDeploymentName = "cilium-operator"
-	hubbleContainerName          = "hubble"
-	hubbleDaemonSetName          = "hubble"
-	hubbleRelayContainerName     = "hubble-relay"
-	hubbleRelayDeploymentName    = "hubble-relay"
-	hubbleUIDeploymentName       = "hubble-ui"
-	redacted                     = "XXXXXX"
+	awsNodeDaemonSetName               = "aws-node"
+	awsNodeDaemonSetNamespace          = "kube-system"
+	ciliumAgentContainerName           = "cilium-agent"
+	ciliumConfigConfigMapName          = "cilium-config"
+	ciliumDaemonSetName                = "cilium"
+	ciliumEtcdSecretsSecretName        = "cilium-etcd-secrets"
+	ciliumOperatorDeploymentName       = "cilium-operator"
+	clustermeshApiserverDeploymentName = "clustermesh-apiserver"
```

Comment:
This is already defined and exported in `defaults/defaults.go`:

https://github.com/cilium/cilium-cli/blob/c43425696d5a902d41c974dea98a6cf39e17e14c/defaults/defaults.go#L51

Some others too, btw. So it might make sense to reuse these consts?

Reply:
Makes total sense, I've added a commit that does that!